brave/brave-core
# fix(wallet): Remove Swap V2 Theme in Nav

## Description 
Removes the `Swap V2` theming from the `Side Nav` to allow the theme to change with the browser theme on the `Swap` page.


Resolves <https://github.com/brave/brave-browser/issues/30498>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and go to the `Swap` tab.
2. Open a new tab and go to brave://settings/appearance
3. Change your browsers theme
4. Go back to the `Swap` tab, the `Side Nav` should change with the new theme.


Before:

https://github.com/brave/brave-core/assets/40611140/08fb335c-81fb-425e-ad0e-70aca9d63f7e

After:

https://github.com/brave/brave-core/assets/40611140/651d4eea-4d01-4612-9ea2-e41791e7afe4
